src/qt/sigmapage.cpp

```diff
@@ -258,6 +258,11 @@ void SigmaPage::on_sendButton_clicked()
     // process sendStatus and on error generate message shown to user
     processSpendCoinsReturn(sendStatus);
 
+    if (sendStatus.status == WalletModel::OK)
```

Comment:
Brace for this one should be the same line.

Reply:
fixed!